src/components/todoItem/index.js

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import styles from "components/todoItem/todoItem.module.css"
+import pencil from "assets/pencil.png"
+import trash from "assets/trash.png"
+import check from "assets/checkmark.png"
+
+function TodoItem({ todo }) {
+  return (
+    <main className={styles.card}>
+      <div className={styles.container}>
+        <h1>{todo.text}</h1>
+        <div className={styles.actions}>
+          <img src={check} alt="Done" className={styles.img} />
+          <img src={pencil} alt="Edit" className={styles.img} />
+          <img src={trash} alt="Delete" className={styles.img} />
```

Comment:
Can we have all the alternative text inside our utils dir? Maybe we can name a file like `constant.js`
And there we will be declaring as well as exporting them and use via import where necessary.

EX: 

`utils/constants.js`

export const COMPLETE_BUTTON_ALT= "Complete Button"

Use them via import

import { COMPLETE_BUTTON_ALT } from "utils/constants.js"
```suggestion
          <img src={check} alt={COMPLETE_BUTTON_ALT} className={styles.img} />
          <img src={pencil} alt={EDIT_BUTTON_ALT} className={styles.img} />
          <img src={trash} alt="{DELETE_BUTTON_ALT} className={styles.img} />
```